From fcb62ce57d5e1a86bfdeced3391321a47250fcce Mon Sep 17 00:00:00 2001
From: Samuli Piippo <samuli.piippo@qt.io>
Date: Tue, 14 May 2019 09:12:42 +0300
Subject: [PATCH] Use correct path to lib dir

-L path was pointing to non-existing directory.

Task-number: QTBUG-75802
Change-Id: I9a72c57ef63d8da860c220041c2000ce3f1fefa8
Reviewed-by: Mikko Gronoff <mikko.gronoff@qt.io>
Reviewed-by: Joerg Bornemann <joerg.bornemann@qt.io>
---
 src/plugins/lipi-toolkit/plugin/plugin.pro | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/plugins/lipi-toolkit/plugin/plugin.pro b/src/plugins/lipi-toolkit/plugin/plugin.pro
index c3a1244..e342af1 100644
--- a/src/plugins/lipi-toolkit/plugin/plugin.pro
+++ b/src/plugins/lipi-toolkit/plugin/plugin.pro
@@ -26,7 +26,7 @@ include(../../../config.pri)
 INCLUDEPATH += \
     ../3rdparty/lipi-toolkit/src/include \
     ../3rdparty/lipi-toolkit/src/util/lib
-LIBS += -L$$OUT_PWD/../../lib \
+LIBS += -L$$OUT_PWD/../../../../lib \
     -lshaperecommon$$qtPlatformTargetSuffix() \
     -lltkcommon$$qtPlatformTargetSuffix() \
     -lltkutil$$qtPlatformTargetSuffix()
